data/society/schools-libraries/index.html

```diff
@@ -1,7 +1,7 @@
 ---
-author:
+  author:
```

Comment:
```suggestion
author:
```
Does this fix downloads?

Reply:
Yes, I just pushed this same commit.

Reply:
💨